package.json

```diff
@@ -74,9 +69,12 @@
     "react-router-dom": "^4.3.1",
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.5",
-    "redux-logger": "^3.0.6",
     "redux-promise-middleware": "^5.1.1",
     "redux-thunk": "^2.3.0",
-    "styled-components": "^3.4.6"
+    "style-loader": "^0.23.0",
+    "styled-components": "^3.4.6",
+    "uglifyjs-webpack-plugin": "^2.0.0",
+    "webpack": "^4.17.2",
+    "webpack-cli": "^3.1.0"
```

Comment:
did we figure it out if this is actually used in the project ?

Reply:
Styled-components: yes alot
uglifyjs: yes in webpack
webpack: yes
webpack-cli: yes to setup webpack. May use it later too